test/integration/kafka_sarama_binding/kafka_test.go

```diff
@@ -91,25 +90,28 @@ func testClient(t testing.TB) sarama.Client {
 }
 
 func testSenderReceiver(t testing.TB) (func(), bindings.Sender, bindings.Receiver, string) {
-	client := testClient(t)
-
 	topicName := "test-ce-client-" + uuid.New().String()
-	p, err := kafka_sarama.NewProtocolFromClient(client, topicName, topicName, kafka_sarama.WithReceiverGroupId(TEST_GROUP_ID))
+
+	clientR := testClient(t)
```

Comment:
nit: please add a code comment or better variable name to indicate what the `R` and `S` suffixes on `client` and `p` mean so our future-selves have an easier time understanding the change :)

Reply:
Done - I can barely remember why/what I did yesterday so this is a good "nit" :-)